src/app/components/timeline/CardDialog.js

```diff
@@ -367,7 +384,9 @@ class CardDialog extends Component {
                 </div>
               </div>
               <div
-                className={cx('card-dialog__details-show-hide-wrapper', {hidden: selected !== 'Description'})}
+                className={cx('card-dialog__details-show-hide-wrapper', {
```

Comment:
is there any way we can make the linter/prettier keep these on one line? It just feels like in this case it makes it harder to read instead of nicer.